src/components/style/interactive/previewCard.css

```diff
@@ -0,0 +1,83 @@
+.fdsCard {
+  color: inherit;
+  text-decoration: none;
+  width: 215px;
+  border-bottom-left-radius: var(--border-radius-default);
+  border-bottom-right-radius: var(--border-radius-default);
+  box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.12), 0 2px 2px 0 rgba(0, 0, 0, 0.24);
+  background-color: var(--color-white);
+}
+
+.fdsCard-cover {
+  height: 130px;
+  position: relative;
+  padding: var(--space-half);
+}
+
+.fdsCard-cover--purple {
+  background-color: var(--color-purple);
+}
+
+.fdsCard-cover--aqua {
+  background-color: var(--color-aqua);
+}
+
+.fdsCard-cover--darkPurple {
+  /* are these one off colors, are these standards, can we replace with standards, etc */
+  background-color: #783983;
+}
+
+.fdsCard-cover--pink {
+  /* are these one off colors, are these standards, can we replace with standards, etc */
+  background-color: #D3346F;
+}
+
+.fdsCard-cover-hover {
+  display: none;
+  text-align: center;
+  padding: 0 var(--space-half);
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: var(--color-charcoal);
+  opacity: 0.95;
+  font-family: var(--font-family-default);
+  color: var(--color-white);
+  letter-spacing: 2px;
```

Comment:
this was `letter-spacing: 1.8`. We had a discussion about this @akdetrick and essentially one of the interesting things here, is if we can have design constrain this letter-spacing to actually be 1px or 2px in Invision inspect, and / or is there value in making this a reusable variable (assuming we continue to use letter-spacing) to further constrain developers and hint that they shouldn't be using non standard values.